tasks/config.yml

```diff
@@ -87,4 +95,29 @@
   template:
      src: bbb/bigbluebutton.j2
      dest: /etc/nginx/sites-available/bigbluebutton
-  notify: reload nginx
+  notify: restart nginx
```

Comment:
One last comment: It is not strictly necessary to completely _restart_ nginx here, a simple reload would be enough...

Reply:
This is the case but not with how BigBlueButton is being installed.

Without restarting nginx I wasn't able to view the BigBlueButton main page and I was instead seeing the demo page.
See also: https://github.com/n0emis/ansible-role-bigbluebutton/pull/3/commits/9d4e884a8ca417e0e7d776cccb426e2c67c5f139

@n0emis Let me know if this work for you.

Reply:
Yes, indeed. I did a installation on a fresh ubuntu 16.04 server, but just with reloading nginx and everything worked fine...

Reply:
Thanks to @robbi5 for pointing out: Restarting is just needed when changing something with the listen directives. But as the default-config already listend on the ports 80 and 443, we do not need to reload here...